### DISCUSSION TITLE
Apple's DHCP Escapement Process

### DISCUSSION THREAD

**Discussion by ghost** [usnistgov/macos_security#256] [Q&A]
I know that the recommendation is to disable captive portal but that seems to have been restricted in Ventura. In my exploration I am seeing confgid and captiveportal and mdns being involved in the DHCP process. Has anyone else seen the same? I know this is a US government forum but I'm curious if anyone is seeing the same thing. I have a firewall rule that I am working on and in my tinkering I am seeing my machine get an assignment for DHCP then open the captive portal app then stop... then configd comes up and then I see some mdns traffic. But something outside of what I'm seeing (and have allowed in my firewall rules) is working in the background. 

Escaprement a mechanism in a clock or watch that alternately checks and releases the train by a fixed amount and transmits a periodic impulse from the spring or weight to the balance wheel or pendulum.

**Comment by robertgendler** [comment]
Where is the recommendation to disable captive portal?

**Reply by ghost** [reply]
You are correct...  I believe this was old NSA guidance... that I still keep in mind.  My apologies. Wrong place. May I delete this thread?

**Reply by robertgendler** [reply]
Sure!
